src/sentry/search/events/builder/spans_indexed.py

```diff
@@ -8,7 +8,6 @@
 
 class SpansIndexedQueryBuilder(QueryBuilder):
     requires_organization_condition = False
-    free_text_key = "span.description"
```

Comment:
Not too sure if i should remove this from the `spans_metrics` dataset or not as well.